frontend/src/App.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './App.css';
+import Home from './containers/home/Home';
+
+const App = () => {
+	return (
+		<div className="App">
+			<Home />
+		</div>
+	);
+};
```

Comment:
Are we using tabs or spaces? If tabs, let's move to 4 spaces. Feel free to ping me if there's an issue.

Reply:
we using tab of width 4 spaces.
### "format": "prettier --print-width 120 --jsx-bracket-same-line --no-bracket-spacing --single-quote --tab-width 4 --use-tabs --write \"**/*.+(js|jsx|css)\""
this is the format script

Reply:
Wait. It uses tabs which isn't right. Use spaces instead.

Reply:
okay i understood and i have made the changes